manager/assets/modext/workspace/package.panels.js

```diff
@@ -147,9 +147,9 @@ Ext.extend(MODx.panel.PackageBeforeInstall, MODx.panel.PackageMetaPanel,{
         Ext.getCmp('package-list-reset').show();
         installBtn.hide().signature = '';
         setupoptionsBtn.hide();
-		if(meta.changelog != undefined){
-			this.addTab(_('changelog'), 'changelog', meta);
-		}
+        if(meta.changelog != undefined){
```

Comment:
```suggestion
        if (meta.changelog !== undefined) {
```

Reply:
Let's do that in 3.x

Reply:
IMO there's nothing wrong with correcting code style on things you're editing in 2.x. As these lines are already marked as changed, correcting the code style wouldn't cause extra conflicts.